### PR TITLE
rootsync package

### PR DESCRIPTION
**Pull request by henderiw**
This is the rootsync package to provision rootsync for a repo.
open point is GIT URL should be mutable one way or another

### CODE REVIEW

File: rootsync/package-context.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kptfile.kpt.dev
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  name: example-rootsync
+  clusterName: example-cluster-name
```
Comment (johnbelamaric) [inline rootsync/package-context.yaml]: missing newline

File: rootsync/rootsync.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name: example-cluster-name
+  namespace: config-management-system
+spec:
+  sourceFormat: unstructured
+  git:
+    repo: http://172.18.0.200:3000/nephio/example-cluster-name.git
+    branch: main
+    auth: token
+    secretRef:
+      name: example-cluster-name-access-token-configsync
```
Comment (johnbelamaric) [inline rootsync/rootsync.yaml]: missing newline

**Review by johnbelamaric — APPROVED** [review]
/approve

non-blocking nits

Agree URL eventually needs to be coming from elsewhere, but this is OK for now

File: rootsync/package-context.yaml (hunk 1 above)
Comment (henderiw) [reply rootsync/package-context.yaml]: fixed

**Pull request merged**